helm_deploy/values-dev.yaml

```diff
@@ -47,6 +47,13 @@ generic-service:
       SecRuleUpdateTargetById 942150 "!ARGS:/outcomeComment/"
       SecRuleUpdateTargetById 942230 "!ARGS:/.*description/"
       SecRuleUpdateTargetById 942410 "!ARGS:/outcomeComment/"
+      # Disable rule 932110 as it only applies to Windows host servers
```

Comment:
There is conflicting advice online about where to place some of these rules - either at the end as here, or directly after the `SecRuleEngine On` - Would welcome any input on this.

Reply:
don't really mind - whatever works.  Best approach is to normally manually edit the ingress in dev to see if the changed rules are working before then raising PR - happy to help with this.

Reply:
note also that any changes here need to also be applied to the preprod cp version as well

Reply:
Thanks @petergphillips will take you up on that offer this afternoon